tests/conftest.py

```diff
@@ -67,9 +67,13 @@ def _ids(resource_type: "Union[ResourceType, str]") -> str:
 @pytest.fixture
 def server() -> "OntoTransServer":
     """Create an `OntoTransServer` test server."""
+    import os
+
     from otelib.ontotransserver import OntoTransServer
```

Comment:
```suggestion
    from otelib.oteclient import OTEClient
```

Reply:
This will be updated once #8 has been merged.